### ISSUE TITLE
Is Xoshiro affecting noise?

### ISSUE THREAD

**Issue by daxpandhi** [dlemstra/Magick.NET#559]
> From the changelog: Replace pseudo-random number generator with a Xoshiro generator.

I wasn't able to find any info on what this affects. Does this affect `SetRandomSeed` or `AddNoise`? Is there an option for forcing the use of the old pseudo-random method instead?

I'm noticing a large difference between 7.14.5 and 7.15.0 when adding noise to a small image (10x10px).

The code where I noticed the change is below. I was using this setup to create a gradient with large fluctuations. With 7.15 these fluctuations show up in different places.

```cs
int x = 10;
mm.Read("gradient:", x, x);            
MagickNET.SetRandomSeed(256);
mm.FilterType = FilterType.Triangle;
mm.AddNoise(NoiseType.Gaussian);
mm.Resize(512, 512);
```

**Comment by dlemstra** [comment]
We discovered a bug in the pseudo-random generator prior to ImageMagick 7.0.9-11, the first 3 values repeated because the random state was not initialized properly. We fixed that and also change the implemenation to a Xoshiro generator. As a consequence of the fix you will get a different numerical sequence when using SetRandomSeed. You might get the result what you want with a different seed. But I still don't understand what kind of effect you are trying to achieve so I cannot really help you with that.
